design-system/ui/modals/QrModal.tsx

```
@@ -54,68 +53,73 @@ const QrModal = ({
+                className="text-11"
+              ></IconText>
+              <IconText
+                size="xSmall"
+                iconPath={<img src={isCheckIn ? qr_check : qr_pending} alt="qr_check" className='mr-1' />}
+                children={isCheckIn ? '체크인 완료' : '체크인 미완료'}
+                className="text-11"
+              ></IconText>
+            </div>
           </div>
+
+          <span onClick={onClick} className="text-deDayTextDark text-xs underline cursor-pointer mt-2">
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)